test/runtests.jl

```diff
@@ -496,24 +610,24 @@ end
 
     ft = ftest(mod, nullmod)
     @test isapprox(ft.pval[1].v,  2.481215056713184e-8)
-    @test sprint(show, ftest(mod, nullmod)) ==
-        """
+    @test all(split(strip(sprint(show, ftest(mod, nullmod)))) .==
```

Comment:
Why would this be needed?

Reply:
When I ran the tests locally, this particular test failed, but only because there was a difference in whitespace. The actual content was the same, but the whitespace was slightly different, so the test failed. I added this code to remove the whitespace and only compare the contents.

Reply:
This caught me off-guard some time ago. Atom/Juno by default uses uses the Whitespace package that adds a trailing line and trims whitespace in every line after each save. I think is fine to have the modification, but maybe add a comment on the reasoning.

Reply:
I have added a comment explaining the reasoning.

Reply:
I don't like this FWIW, if editors are broken they should be fixed instead of making our code more complex. Doctests like this are common in Julia, we really don't want to add comments explaining this every time they occur.